edge-util/src/Microsoft.Azure.Devices.Edge.Util/edged/version_2018_06_28/generatedCode/HttpWorkloadClient.cs

```diff
@@ -555,7 +555,7 @@ public async System.Threading.Tasks.Task<CertificateResponse> CreateServerCertif
                         ProcessResponse(client_, response_);
     
                         var status_ = ((int)response_.StatusCode).ToString();
-                        if (status_ == "201") 
+                        if (status_ == "200") 
```

Comment:
@and-rewsmith WorkloadClientTest.GetServerCertificatesFromEdgeletShouldReturnCert failed if 201 is used here.  I found that master use 200,  do you know why generated code changed to 201?

Reply:
No, the only difference is that I generated with the dotnet 3 option. We should find a way to document these changes so that the next time someone generates the code they don't face the same headache.

Reply:
We should also change on line 464, right?

Reply:
I am thinking we should update server side to return 201 instead of 200.  Would you please have to work on this.  I think I should revert this change, and handle this in another PR.  What do you think?

Reply:
I agree with separate PR and can help make this change on the server side (edgelet). I think we should first prioritize the CI, e2e, and release build though.

Reply:
Reverted.